PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -626,6 +626,8 @@ def miniAOD_customizeData(process):
     process.load("RecoPPS.Local.ctppsLocalTrackLiteProducer_cff")
     process.load("RecoPPS.ProtonReconstruction.ctppsProtons_cff")
     process.load("Geometry.VeryForwardGeometry.geometryRPFromDB_cfi")
+    process.load('L1Trigger.L1TGlobal.simGtExtFakeProd_cfi')
+    process.simGtExtFakeProd.tcdsRecordLabel = cms.InputTag("tcdsDigis","tcdsRecord")
     task = getPatAlgosToolsTask(process)
```

Comment:
```suggestion
    process.simGtExtFakeProd.tcdsRecordLabel = ("tcdsDigis","tcdsRecord")
    task = getPatAlgosToolsTask(process)
    task.add(process.simGtExtFakeProd)
```
-  please drop type specifications for all parameters which already exist. This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.
- the module should be added to an active task to be enabled